### DISCUSSION TITLE
Multiplayer currently playing map name beaks line for each word

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#15449] [Q&A] [answered]
This is not intended, isn't it?

![image](https://user-images.githubusercontent.com/43073074/140057416-4535ace5-f74b-4d96-ba5a-931b2a93f90e.png)

![image](https://user-images.githubusercontent.com/43073074/140057548-ca7218dc-c36b-4434-86bc-78806dcea6ac.png)

**Comment by bdach** [comment, marked as answer]
Regressed with text flow changes due to the hacky 0.5 relative width spec there... I'll fix when I get off work.

**Reply by bdach** [reply]
https://github.com/ppy/osu/issues/15451
